### ISSUE TITLE
Always applied same DDL with decimal default value

### ISSUE THREAD

**Issue by mkanenobu** [k0kubun/sqldef#85]
## Problem
As a title

on mysqldef: v0.8.4 linux amd64 (maybe latest)

## Expected
Not apply any changes

### Initial schema

```
CREATE TABLE `some_table` (
  `some_value` decimal(5, 2) DEFAULT 0.0
);
```

### Input DDLs

Same as initial schema

```
CREATE TABLE `some_table` (
  `some_value` decimal(5, 2) DEFAULT 0.0
);
```

### Output DDLs

```
ALTER TABLE some_table CHANGE COLUMN some_value some_value decimal(5, 2) DEFAULT 0.000000;
```

**Comment by k0kubun** [comment]
Thanks, I fixed it in v0.8.5.

**Comment by mkanenobu** [comment]
Thanks!!
